### DISCUSSION TITLE
Error 404 trying to use statamic control panel

### DISCUSSION THREAD

**Discussion by AlissaVal** [statamic/cms#4928] [Q&A] [answered]
Hi, i recently just started using statamic for local use. I first had a lot of trouble setting up the .test cite. After 3 days of troubleshooting i finally got it to work with my localhost. Now after i made my super user, i go to the control panel to sign in and i get a 404. what should i do?

**Comment by jasonvarga** [comment, marked as answer]
What kind of web server are you using?

**Reply by AlissaVal** [reply]
IIS windows 10

**Reply by belisards** [reply]
Hi, I have the same issue with a Linux + Apache webserver. It is a fresh installation using statamic-cli and I can access the landing page, but `/cp` gives me a 404 error. Any tip here? @jasonvarga

**Reply by duncanmcclean** [reply]
Can you share a screenshot of the 404 page you're seeing?

**Reply by belisards** [reply]
Sure, @duncanmcclean, thank you for your response!

Here it is: https://imgur.com/a/wKTjOdG

I am using the following environment:

Ubuntu 21.10
PHP 8.1.2 (cli) 
Apache/2.4.48
Composer version 2.2.6

**Reply by duncanmcclean** [reply]
Looks like a server issue to me. Your URL rewrites aren't working.

In your public folder, do you have a .htaccess file?

**Reply by belisards** [reply]
I had, but there were two problems.

1. mod_rewrite was not enabled: so I enabled it with `sudo a2enmod rewrite`
2. After that, I got this error in my Apache log "Option MultiViews not allowed here", so I have removed the option `-MultiViews` from the default `.htaccess`

Now, I can see the control panel. Many thanks @duncanmcclean!

**Reply by duncanmcclean** [reply]
Fantastic - no worries!
